### PR TITLE
Update allows editing setting

### PR DESCRIPTION
**Pull request by Jmats17**
-Update AssetsCameraManager to default allowsEditing to false & add a function that allows updating in future.
-Add a boolean property to AssetsPickerConfig to allow developer using the library the ability to set the flag in other scenarios where editing can be allowed.
-Update the AssetsPhotoViewController which holds a reference to both the manager and config to update the allowsEditing flag

### CODE REVIEW

File: AssetsPickerViewController/Classes/Photo/Controller/AssetsPhotoViewController.swift [hunk 1]
```diff
@@ -161,6 +161,8 @@ open class AssetsPhotoViewController: UIViewController {
                 self.delegate?.assetsPickerCannotAccessPhotoLibrary?(controller: self.picker)
             }
         }
+      
+        cameraPicker.setAllowsEditing(shouldAllow: pickerConfig.allowsEditing)
```
Comment (alanipce) [inline AssetsPickerViewController/Classes/Photo/Controller/AssetsPhotoViewController.swift:165]: is it safe to assume a picker config exists are this point? seems other uses of it optionally bind

File: AssetsPickerViewController/Classes/Picker/AssetsPickerConfig.swift [hunk 2]
```diff
@@ -128,6 +128,11 @@ open class AssetsPickerConfig : NSObject {
         return CGSize(width: edge, height: edge)
     }
     
+    // MARK: - Manager Settings
+  
+    /// Flag that determines if a user can crop a selected photo
+    open var allowsEditing: Bool = false
```
Comment (alanipce) [inline AssetsPickerViewController/Classes/Picker/AssetsPickerConfig.swift]: seems like the old default was true and we want to be able to set to false

**Review by alanipce — COMMENTED** [review]
Can you add .DS_Store to the gitignore file to avoid all these new files being committed

File: AssetsPickerViewController/Classes/Photo/Controller/AssetsPhotoViewController.swift (hunk 1 above)
Comment (Jmats17) [reply AssetsPickerViewController/Classes/Photo/Controller/AssetsPhotoViewController.swift:165]: @alanipce I think so yes. There is only one spot the AssetsPhotoViewController gets initialized and immediately after a non-optional AssetsPickerConfig is passed to the assetphoto vc's picker config property. So I think it is safe to assume that a picker config exists inside the viewDidLoad. 

Additionally, looking at the extension file with +Setup, those functions (such as setupBarButtonItems) are also directly calling on pickerConfig w/out unwrapping so I think we should be good.

**Review by alanipce — APPROVED** [review]

**Pull request merged**